### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by rustamwin**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ✔️
| Fixed issues  | -

### CODE REVIEW

**Comment by rustamwin** [comment]
[Yii-auth-client refactoring](https://trello.com/c/lgRdZGFs/82-yii-auth-client-refactoring)

**Comment by samdark** [comment]
@rustamwin master should be merged into this branch before furhter cleanup.

**Comment by samdark** [comment]
@rustamwin is this one ready to review/merge it?

File: src/AbstractOpenId.php [hunk 1]
```diff
@@ -33,7 +33,7 @@
  *
  * @link http://openid.net/
  */
-class OpenId extends BaseClient
+abstract class AbstractOpenId extends AbstractAuthClient
```
Comment (samdark) [inline src/AbstractOpenId.php:36]: Let's not prefix with `Abstract`. See https://github.com/yiisoft/docs/blob/master/010-code-style.md#abstract-classes

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
Code looks alright but tests aren't OK. Would you please fix these before merge?

**Comment by samdark** [comment]
👍

**Pull request merged**